.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "bracketSpacing": false,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "arrowParens": "avoid"
+}
```

Comment:
This configuration uses Google's typescript style guide.

Reply:
As I said with using double quotes, the style is a preference.

Reply:
It doesn't really matter anyway, as long as you are consistent. :)

Reply:
Of course lol. Appreciate it